carbonserver/tests/api/test_api.py

```diff
@@ -0,0 +1,109 @@
+import random
+
+import requests
+
+URL = "http://localhost:8008"
+experiment_id = project_id = user_id = api_key = org_id = team_id = email = None
+password = "Secret1!îstring"
+
+
+def test_create():
+    global user_id, api_key, org_id, team_id, email
+    email = f"test-{random.randint(1, 1000)}@test.com"
+    payload = {"email": email, "name": "toto", "password": password}
+    r = requests.post(url=URL + "/users/signup/", json=payload, timeout=2)
+    print(r.json())
+    assert r.status_code == 201
+    assert r.json()["email"] == email
+    assert r.json()["is_active"] == True  # noqa
+    user_id = r.json()["id"]
+    api_key = r.json()["api_key"]
+    org_id = r.json()["organizations"][0]
+    team_id = r.json()["teams"][0]
+
+
+def test_auth_success():
+    payload = {"email": email, "password": password}
+    r = requests.post(url=URL + "/authenticate/", json=payload, timeout=2)
+    print(r.json())
```

Comment:
we could log payloads for important tests to see payloads even if the test is green, but prints should be removed